apps/condo/domains/contact/schema/Contact.js

```diff
@@ -159,6 +158,16 @@ const Contact = new GQLListSchema('Contact', {
             }
         },
     },
+    kmigratorOptions: {
+        constraints: [
+            {
+                type: 'models.UniqueConstraint',
+                fields: ['property', 'unitName', 'unitType', 'phone'],
```

Comment:
probably, we need to add some test for that case

Reply:
we have such tests in `Contact.test.js` in `unique constraint` describe block